host/matching_simulation_test.go

```diff
@@ -176,31 +181,35 @@ func (s *MatchingSimulationSuite) TestMatchingSimulation() {
 	numPollers := getNumPollers(s.testClusterConfig.MatchingConfig.SimulationConfig.NumPollers)
 	pollDuration := getPollDuration(s.testClusterConfig.MatchingConfig.SimulationConfig.PollTimeout)
 	polledTasksCounter := int32(0)
+	maxTasksToGenerate := getMaxTaskstoGenerate(s.testClusterConfig.MatchingConfig.SimulationConfig.MaxTaskToGenerate)
+	var tasksToReceive sync.WaitGroup
+	tasksToReceive.Add(maxTasksToGenerate)
```

Comment:
Simulation runs for 180s at most ([timeout set here](https://github.com/uber/cadence/blob/06e5a6dc9ec0715298de27e6024234fcddf47656/docker/buildkite/docker-compose-local-matching-simulation.yml#L30)) and `maxTasksToGenerate` might not be hit if it's set too high. please add a comment in the test code here about this to help identify "simulation is stuck" cases due to misconfiguration

Reply:
Done, thanks.